tutorials/tour/_posts/2017-02-13-operators.md

```diff
@@ -10,8 +10,40 @@ num: 30
 next-page: automatic-closures
 previous-page: local-type-inference
 ---
+In Scala, operators are methods. For example, `+` can be called with dot-notation:
+```
+10.+(1)
+```
+
+However, it more closely resembles math if you write it as an infix operator:
```

Comment:
I don't think "closely resembles math" is quite what you mean.  It *is* math.  It more closely resembles classical mathematical notation for addition, but that's quite a mouthful.

Maybe something like, "However, the expression is easier to recognize when written in a familiar form with `+` as an operator:".  (I don't think that wording is ideal either, actually.)